src/helpers/get-url-parameter.js

```diff
@@ -6,9 +6,16 @@
  * @returns {String}    param value if it's available
  */
 export default function getUrlParameter(name) {
-  name = name.replace(/[[]/, '\\[').replace(/[\]]/, '\\]');
-  const regex = new RegExp(`[\\\\?&]${name}=([^&#]*)`);
-  const results = regex.exec(window.location.search);
+  if (!name) {
+    console.error('There is no param set in the URL');
 
-  return results === null ? '' : decodeURIComponent(results[1].replace(/\+/g, ' '));
+    return null;
+  }
+
+  name = name.replace(/[[]/, '\\[')
+    .replace(/[\]]/, '\\]');
```

Comment:
todo: I think this is a project related, specific use case, that we have a square bracket after the param name. Please delete this helper as well and use the following code where this helper is currently used:

```
  const urlParam = new URLSearchParams(window.location.search);
  const themeId = urlParam.get('theme');
```

Maybe you can even use a one liner:

```
const themeId = new URLSearchParams(window.location.search).get('theme');
```